LabExT/Movement/MoverNew.py

```diff
@@ -442,6 +445,39 @@ def _lift_lower_stages(lift):
         finally:
             _lift_lower_stages(-self.z_lift)
 
+    @assert_connected_stages
+    def move_to_device(self, chip: Type[Chip], device_id: Any):
+        """
+        Moves stages to device.
+
+        Parameters
+        ----------
+        chip: Chip
+            Instance of a imported chip.
+        device_id: Any
+            Device ID to which the stages should move.
+
+        Raises
+        ------
+        MoverError
+            If no device was found for the given ID.
+        """
+        device = chip._devices.get(device_id)
+        if device is None:
+            raise MoverError(
+                f"No device was found for the given chip for the given device ID {device_id}.")
+
+        movement_commands = {}
+        if self.input_calibration:
+            movement_commands[self.input_calibration.orientation] = device.input_coordinate + \
+                ChipCoordinate(z=self.z_lift)
```

Comment:
Why do you add the z_lift coordinate to the target coordinate to move to? This does not make any sense as we want to exactly arrive at coordinates (x, y, 0), no?

z-lift is only temporary during movement